### PR TITLE
[Play] Multi-Select Component

### LINKED ISSUE #690: [Add a Play]: Multi-select Component
Thank You! 

## What's the Play about?
A Select component that allows the users to choose multiple values.

## What ReactJS concept will be used to create this play?
React hooks ( useState, useEffect )

## Are you willing to take it up for implementation?
Yes

### PR DESCRIPTION
**Pull request by Abrar0204**
# Description

This play is a select component that allows the user to choose multiple values and also search through them.

Fixes #690 

## Type of change

- [x] New feature (non-breaking change which adds functionality) 

# Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by koustov** [comment]
@ammaaraslam can you look into this PR?

**Review by ammaaraslam — CHANGES_REQUESTED** [review]
@Abrar0204
Few thoughts on the play...
- Currently you can click on any selected item to remove it - I suggest to add a remove button inside each item so that it's obvious to the user that they can remove an item from the selection.
- Another, would be that after you search for something that's not there, it'd be a good idea to display a message like _"Game not found"_.
- And since the content doesn't utilize much space, I'd suggest you to increase the sizing of it so that it doesn't look too blank.
- And finally, I see that the user cannot add a game that is not in the initial list. Is that intentional?

Apart from these minor things, everything else looks great!
@Abrar0204 let me know your thoughts on these.

**Comment by Abrar0204** [comment]
Thanks for the comments @ammaaraslam. I'll make the changes suggested in points 1,2,3 and as for adding a game to the list, I intended for the play to be a standalone select component so I did not feel the need to add the ability to add your own games. But if we go about adding that feature, I think introducing an input field above the select component to add your games would be good. What do you think?

**Comment by ammaaraslam** [comment]
> Thanks for the comments @ammaaraslam. I'll make the changes suggested in points 1,2,3 and as for adding a game to the list, I intended for the play to be a standalone select component so I did not feel the need to add the ability to add your own games. But if we go about adding that feature, I think introducing an input field above the select component to add your games would be good. What do you think?

Awesome 🚀 
If you're including the feature of adding own games, a separate input component won't be necessary.
Anyways a user will search for a game, and if it's not there just display a "Game not found" message and a button that says "Add game to list" or so in the dropdown itself. And once the user clicks on that button, the input from the search is put into the list of games.
It's just like what you see in GitHub for something like creating a branch.

Or you can go your route if that seems like a better and easier choice to you.

**Review by ammaaraslam — APPROVED** [review]
@Abrar0204 All good can merge!

File: src/plays/multiselect-component/styles.scss [hunk 1]
```diff
@@ -0,0 +1,100 @@
+$select-component-width: 1000px;
+$select-component-font-size: 25px;
+$select-component-dropdown-max-height: 450px;
+
+.multi-select-container {
+  display: flex;
+  justify-content: center;
+}
+
+#select {
```
Comment (koustov) [inline src/plays/multiselect-component/styles.scss:10]: do not use the reserved keywords as selectors. Use play specific name instead like '#multi-select-container-select`

**Review by koustov — CHANGES_REQUESTED** [review]
one tiny comment. rest looks good

File: src/plays/multiselect-component/styles.scss (hunk 1 above)
Comment (Abrar0204) [reply src/plays/multiselect-component/styles.scss]: done

**Pull request merged**